internal/gatewayapi/runner/runner.go

```diff
@@ -97,6 +99,9 @@ func (r *Runner) subscribeAndTranslate(ctx context.Context) {
 				}
 				// Translate to IR
 				result := t.Translate(resources)
+				if result.LoggedErrors != nil {
```

Comment:
changing the `Translate` signature to return `result, err` and logging that would be more idiomatic

Reply:
I disagree.

The idiomatic nature would require that the error returned from `Translate` be treated as an actual error, since the caller can't distinguish between aggregated errors that are really errors and aggregated errors that should just be logged.

If the `Translate` signature changes to include an error, the `Runner`  should then change to treat the returned error _as an error_. Just logging a returned error and then ignoring it by continuing would appear like a bug.

If, in the future, there are show-stopping errors that would be returned from the `Translate` function, it would be impossible to distinguish those errors that should be handled by the `Translate` callers in some concrete way, and those that we just want printed to the log right now.

Reply:
this is a long running go routine / runner, we only want log errors, not return or shut down, and track them using telemetry metrics. 
There is currently no err object we are returning, so there is no distinction of errors, there are just logged errors